pykechain/models/widgets/helpers.py

```diff
@@ -107,6 +108,200 @@ def _set_title(
     return meta, title
 
 
+def _set_description(
+        meta: Dict,
+        description: Optional[Union[Text, bool]] = None,
+        **kwargs
+) -> Tuple[Dict]:
+    """
+    Set the customDescription in the meta based on provided optional custom description or no description.
+
+    This will inject into the meta the `customDescription` and `showDescriptionValue` if the `description` is provided
+    as argument, otherwise it will inject 'no description'. It returns the meta definition of the widget.
+
+    :param meta: meta dictionary to augment
+    :type meta: dict
+    :param description: A description for the widget
+            * String value: use the description
+            * None or False: No description at all.
+    :type description: basestring or None or bool
+
+    :return: tuple of meta
+    :rtype: Tuple[Dict,Text]
+    :raises IllegalArgumentError: When description is neither str, bool or None.
+    """
+    if description is False or description is None:
+        show_description_value = "No description"
```

Comment:
this could use some enums, or something similar. Maybe not in the main enums, but `widget_enums.py` or something?

Reply:
Made it in enums for now as there we have all the enums. But good point, we need to refactor this one day carefully to make sure backwards compatibility is maintained.